.travis.yml

```diff
@@ -16,4 +17,5 @@ deploy:
   password: $PYPI_PASSWORD
   distributions: "sdist bdist_wheel"
   on:
+    python: 3.7
```

Comment:
Travis' default dist action w/pypi fails with an error on python 3.6 now, so I changed this to publish with 3.7